README.md

```diff
@@ -1,25 +1,32 @@
-# relay-workshop
+# Relay Learning Exercises
 
-Home for Artsy's [Relay](https://relay.dev/docs/v10.1.3/) peer learning tutorial.
+[Relay](https://relay.dev/docs/v10.1.3/) is a powerful GraphQL client. At Artsy we use it in both our [web app](https://github.com/artsy/force) and [mobile app](https://github.com/artsy/eigen). 
 
-## Why?
-
-Relay is a powerful GraphQL client, but it's complex and difficult to learn. These exercises aim to ease your learning by introducing Relay concepts a little bit at a time. 
+Relay is also complex and difficult to learn. This project contains exercises to help you learn Relay[*](#which-version-of-relay-are-we-learning) by introducing concepts a little bit at a time. 
```

Comment:
> Relay is also complex and difficult to learn.

maybe add a `but, once you've learned the conventions it becomes straightforward and you'll never want to go back!` (or something to that effect) -- meaning, we can lighten up the mood a bit?

Reply:
Addressed in https://github.com/artsy/relay-workshop/pull/25/commits/c3e5ef4050c7cf4537e746ccf8e82e16cf7d4af4.